.github/check_quote_includes.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v rg)" ]; then
+   GREP="grep -R"
+else
+   GREP="rg -j 1 --no-heading -N"
+fi
+
+${GREP} "#include \"vecmem" "$@" ; test $? -eq 1 || exit 1
+${GREP} "#include \"detray" "$@" ; test $? -eq 1 || exit 1
+${GREP} "#include \"Acts" "$@" ; test $? -eq 1 || exit 1
+${GREP} "#include \"covfie" "$@" ; test $? -eq 1 || exit 1
+${GREP} "#include \"algebra" "$@" ; test $? -eq 1 || exit 1
```

Comment:
🤔 I think you're right, this is probably the most reasonable way to implement it. Technically it would be better to do this in a way that checks whether a file is part of this repository (to allow local subdirectories called `"vecmem"` for instance), but it's not worth putting that much effort into this. 🤔

Reply:
Mind you pre-commit does this for us already, so only files which we want to check end up in `$@`.

Reply:
(See https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includecategories)

Reply:
As discussed, I meant a slightly different thing here. But the functionality in clang-format I didn't know about, that was a good pointer.